src/game.js

```diff
@@ -35,6 +49,15 @@ module.exports = class Game{
         return roleList;
     }
 
+    saveGame(game)
```

Comment:
Could this be called in the constructor? Like `this.saveGame();` so we could call `gameList.push(this);`